rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/TemplateUtils.java

```diff
@@ -60,6 +60,41 @@ protected Path downloadArtifactToTmpFile(BakeManifestEnvironment env, Artifact a
     return path;
   }
 
+  protected void downloadArtifactToTmpFileStructure(BakeManifestEnvironment env, Artifact artifact) throws IOException {
+    if (artifact.getReference() == null) {
+      throw new InvalidRequestException("Input artifact has an empty 'reference' field.");
+    }
+    Path githubPath =  Paths.get(artifact.getReference().substring(
+            artifact.getReference().indexOf(PATH_SEPARATOR)+PATH_SEPARATOR.length()));
```

Comment:
The `name` property of the artifact should give us this data. According to the docs, the `name` will be the path to the file in the repo so we can use that to get our base and subfolders.

Reply:
https://www.spinnaker.io/reference/artifacts/types/github-file/

Reply:
removed this logic due now the temporal files are going to be saved based on their full path.